src/models/GraphData.js

```diff
@@ -242,6 +254,28 @@ export default class GraphData {
     return transformedItem;
   }
 
+  /* eslint-disable class-methods-use-this */
+  transformFoodResponseDataItem(item) {
```

Comment:
Perhaps we should be using `lodash.get` for these kind of nested property checks.  We use it extensively in our other repos as a standard practice.
```
let transformedItem;
const value = _.get(item, 'nutrition.carbohydrate.net');
if (value) {
  const time = parse(item.time).getTime() / 1000;
  transformedItem = {
    time,
    value,
  };
}
```

Reply:
Thanks Clint. If it's ok, how about ES6 restructuring with defaults instead. Once you get used to it it's pretty readable. And it's a lot faster than _.get, and maybe more idiomatic. Ideally we'd have null-coalescing/optional-chaining, like in Swift (that's coming to JavaScript!), but, in lieu of that, how about not taking the lodash hit at this point (yet) for this one-off use case in the React Native codebase. I'll push a commit that fixes in this way.